rest-api/src/main/resources/definitions/enums/participant_study_progress.yml

```diff
@@ -0,0 +1,13 @@
+type: string
+description: |
+    They type of notification to provide relative to a session’s time window. This enum specifies when the notification should be shown to the participant.
```

Comment:
They -> The

Reply:
Comment is totally wrong at the top...I've rewritten this for the specific comment.